.github/workflows/sv_pipeline_docker.yml

```diff
@@ -3,15 +3,15 @@ name: Docker Images
 on:
   push:
     branches:
-      - master
+      - main
```

Comment:
Did you mean to disable this for now, until we've changed the image tag handling?